src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/IdentifyRasterCell/IdentifyRasterCell.xaml

```diff
@@ -0,0 +1,8 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.IdentifyRasterCell.IdentifyRasterCell"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:MapView x:Name="MyMapView"/>
```

Comment:
I think the style guide requires a space between the `"` and the `/`, but you might want to double check that.